src/events/message/commands/help/link.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+import { CommandArg, Message, word } from "../types";
+
+// "?link <topic>" provides a simple shortcut to various useful URLs
+
+const LINKS: { [k: string]: string } = {
+  docs: "https://docs.codewars.com/",
+  honor: "https://docs.codewars.com/gamification/honor",
+  katas: "https://www.codewars.com/kata/latest/my-languages",
+  authoring: "https://docs.codewars.com/authoring",
+  ranking: "https://docs.codewars.com/curation/kata",
+  troubleshooting: "https://docs.codewars.com/training/troubleshooting",
+  github: "https://github.com/codewars/codewars.com/issues",
```

Comment:
I might want `github` mapped to `https://github.com/codewars`, but we can adjust these later.